modules/file-system/pre-existing-network-storage/scripts/install-daos-client.sh

```diff
@@ -27,29 +36,15 @@ else
 	# The following commands should be executed on each client vm.
 	## For Rocky linux 8 / RedHat 8.
 	if [ "${OS_ID}" = "rocky" ] || [ "${OS_ID}" = "rhel" ]; then
-		if [ "${OS_VERSION_MAJOR}" = "8" ]; then
-			# 1) Add the Parallelstore package repository
-			cat >/etc/yum.repos.d/parallelstore-v2-6-el8.repo <<EOF
-[parallelstore-v2-6-el8]
-name=Parallelstore EL8 v2.6
-baseurl=https://us-central1-yum.pkg.dev/projects/parallelstore-packages/v2-6-el8
-enabled=1
-repo_gpgcheck=0
-gpgcheck=0
-EOF
-		elif [ "${OS_VERSION_MAJOR}" -eq "9" ]; then
-			cat >/etc/yum.repos.d/parallelstore-v2-6-el9.repo <<EOF
-[parallelstore-v2-6-el9]
-name=Parallelstore EL9 v2.6
-baseurl=https://packages.daos.io/v2.6/EL9/packages/x86_64/
-enabled=1
-repo_gpgcheck=0
-gpgcheck=0
-EOF
-		else
-			echo "Unsupported RedHat / Rocky Linux system version ${OS_VERSION_MAJOR}. This script only supports version 8 and 9."
-			exit 1
-		fi
+		# 1) Add the Parallelstore package repository
+		cat >/etc/yum.repos.d/parallelstore-v2-6-el"${OS_VERSION_MAJOR}".repo <<-EOF
+			[parallelstore-v2-6-el${OS_VERSION_MAJOR}]
+			name=Parallelstore EL${OS_VERSION_MAJOR} v2.6
+			baseurl=https://us-central1-yum.pkg.dev/projects/parallelstore-packages/v2-6-el${OS_VERSION_MAJOR}
```

Comment:
baseurl is different for rhel-9

baseurl=https://packages.daos.io/v2.6/EL9/packages/x86_64/

Reply:
Maybe that was true at one point but our documentation shows the same URL pattern:

- https://cloud.google.com/parallelstore/docs/connect-from-compute-engine#rocky-linux-9-optimized_1
- https://cloud.google.com/parallelstore/docs/connect-from-compute-engine#rhel-9_1

For RHEL9 derivatives

```
baseurl=https://us-central1-yum.pkg.dev/projects/parallelstore-packages/v2-6-el9
```

and for RHEL8 derivatives:

```
baseurl=https://us-central1-yum.pkg.dev/projects/parallelstore-packages/v2-6-el8
```

Reply:
Thanks for explaining. Seems this is updated recently.